docs/detections/rules-ui-manage.asciidoc

```diff
@@ -24,9 +24,7 @@ On the Rules page, you can:
 [[sort-filter-rules]]
 === Sort and filter the rules list
 
-To sort the rules list, click the *Rule*, *Last updated*, or *Enabled* column header.
-
-TIP: To sort by any other column, turn on the *Advanced sorting* toggle above the table. If you experience performance issues when working with the table, you can turn this setting off.
+To sort the rules list, click the *Rule*, *Last updated*, or any other column header. To sort in descending order, click the column header again.
```

Comment:
It might be ok to just say that users can sort by any column header, for example:

```suggestion
To sort the rules list, click any column header. To sort in descending order, click the column header again.
```